cms/jinja2/templates/pages/methodology_page.html

```diff
@@ -98,10 +106,7 @@
                         "title": toc_title,
                         "ariaLabel": toc_aria_label,
                         "itemsList": table_of_contents,
-                        "relatedLinks": {
-                            "title": related_publications_title,
-                            "itemsList": related_publications_list
-                        },
+                        "relatedLinks": [related_publications_item] | selectattr('title') | list,
```

Comment:
I would rather make the change in `MethodologyPage.get_formatted_related_publications_list`

or

```jinja2
{% set related_publications_items = [{
                                "title": related_publications_title,
                                "itemsList": related_publications
                            }]
                        %}
```

and `"relatedLinks": related_publications_items`

Reply:
update:

"relatedLinks" is a dict, so `"relatedLinks": related_publications_items` is the alternative

Reply:
Thank you Dan for your infinite wisdom as always 🙏